### PR TITLE
Use web app user in logs and process management

### LINKED ISSUE #111: Use logged in web app user in drunc auth token
Currently drunc is running using a Dummy Authoriser which doesn't actually check the [auth tokens](https://github.com/DUNE-DAQ/druncschema/blob/develop/schema/druncschema/token.proto) used to make gRPC calls. Drunc does however use the user_name value associated with the token.

When we make gRPC calls, we're currently generating tokens using [create_dummy_token_from_unname](https://github.com/DUNE-DAQ/drunc/blob/008c6024cd0c74dfe24e1ecc555752fa6ad5019c/src/drunc/utils/shell_utils.py#L314). This asks the OS for the username of the currently logged in system user (nobody in the case of the docker image). Drunc logs and Kafka messages then show that the system user is performing actions. It would be nicer if the username used in the token corresponded to the logged in user of the web app. It's pretty easy to role our own tokens for this.

### PR DESCRIPTION
**Pull request by TinyMarsh**
# Description
This PR adresses #111 by passing the web app username - from the `request` object available in views - to function signatures where the username is relevant to the function i.e. logs and process management.

Fixes #111

## Type of change

- [ ] Documentation (non-breaking change that adds or improves the documentation)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (non-breaking, back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (whatever its nature)

## Key checklist

- [ ] All tests pass (eg. `python -m pytest`)
- [ ] The documentation builds and looks OK (eg. `python -m sphinx -b html docs docs/build`)
- [ ] Pre-commit hooks run successfully (eg. `pre-commit run --all-files`)

## Further checks

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added or an issue has been opened to tackle that in the future. (Indicate issue here: # (issue))

### CODE REVIEW

**Review by cc-a — CHANGES_REQUESTED** [review]
Looks good. One tiny tweak so that our dummy processes can keep running.

**Review by dalonsoa — APPROVED** [review]
Keeping Chris comment aside, this looks good to me!

File: process_manager/process_manager_interface.py [hunk 1]
```diff
@@ -6,32 +6,33 @@
...
     ProcessUUID,
 )
+from druncschema.token_pb2 import Token
 
 
-def get_process_manager_driver() -> ProcessManagerDriver:
+def get_process_manager_driver(username: str) -> ProcessManagerDriver:
     """Get a ProcessManagerDriver instance."""
-    token = create_dummy_token_from_uname()
+    token = Token(token=f"{username}-token", user_name=username)
```
Comment (AdrianDAlessandro) [inline process_manager/process_manager_interface.py:21]: Assuming this actually works with their authentication system

**Review by AdrianDAlessandro — APPROVED** [review]
LGTM!

File: process_manager/process_manager_interface.py (hunk 1 above)
Comment (cc-a) [reply process_manager/process_manager_interface.py:21]: For the moment they're using a `DummyAuthoriser` that doesn't check the token.

**Review by cc-a — APPROVED** [review]

**Pull request merged**